axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -58,44 +23,40 @@
           "codehash": "0xf0ad66defbe082df243d4d274e626f557f97579c5c9e19f33d8093d6160808b7",
           "predeployCodehash": "0x73fc31262c4bad113c79439fd231281201c7c7d45b50328bd86bccf37684bf92",
           "salt": "Create3Deployer"
+        },
+        "Operators": {
+          "owner": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
+          "address": "0x60607ebAE692cfD36af19b4779b72AC2f1Ca2798",
+          "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
+          "deploymentMethod": "create2",
+          "codehash": "0xc561dc32ef670c929db9d7fbf6b5f6c074a62a30602481ba3b88912ca6d79feb",
+          "predeployCodehash": "0xc561dc32ef670c929db9d7fbf6b5f6c074a62a30602481ba3b88912ca6d79feb",
+          "salt": "Operators devnet-amplifier(core)"
+        },
+        "AxelarGasService": {
+          "collector": "0x60607ebAE692cfD36af19b4779b72AC2f1Ca2798",
+          "address": "0xc429E38e4F47B6c8c937fe12a5E1F58Bff785228",
```

Comment:
can you redeploy the gas service for core chains with `-m create2` and same salt as Operators contract? This way the address derivation matches. Subsequent upgrades can use any deploy method since the proxy is already fixed (as we did for amplifier gas service)